Gemfile

```diff
@@ -6,6 +6,27 @@ else
   gem 'puppet', :require => false
 end
 
+gem 'metadata-json-lint'
 gem 'puppetlabs_spec_helper', '>= 0.1.0'
```

Comment:
This should be 1.1.1 and then the `bundle exec metadata-json-lint metadata.json` from .travis.yml can be removed